packages/collector/Collector.php

```diff
@@ -0,0 +1,229 @@
+        collector_zip_send();
+        collector_zip_delete();
+        exit();
+    }
+}
+
+function collector_plugin_menu()
+{
+    add_submenu_page(
+        NULL,
+        'Collector',
+        'Collector',
+        'manage_options',
+        'collector_render_playground_page',
+        'collector_render_playground_page',
+        NULL
+    );
+}
+
+function collector_render_playground_page()
+{?>
+	<div id = "wp-playground-wrapper">
+		<div id = "wp-playground-toolbar">
+			NOW WORKING INSIDE WORDPRESS PLAYGROUND &nbsp; [<a href = "/wp-admin" id = "goBack">RETURN</a>]
+		</div>
+		<div id = "wp-playground-main-area">
+			<iframe id = "wp-playground"></iframe>
+		</div>
+	</div>
+	<script type = "text/javascript">
+		const frame  = document.getElementById('wp-playground');
+		const zipUrl = <?=json_encode(COLLECTOR_DOWNLOAD_PATH);?>;
+
+		const username   = <?=json_encode(wp_get_current_user()->user_login);?>;
+		const fakepass   = <?=json_encode(collector_get_fakepass());?>;
+		const pluginUrl  = new URLSearchParams(window.location.search).get('pluginUrl');
+        const pluginName = new URLSearchParams(window.location.search).get('pluginName');
+
+		(async () => {
+			const  { startPlaygroundWeb } = await import(<?=json_encode(COLLECTOR_PLAYGROUND_PACKAGE);?>);
+
+			const steps = [
+				{
+					step: 'writeFile',
+					path: '/data.zip',
+					data: {
+						'resource': 'url',
+						'url': zipUrl,
+					},
+				},
+				{
+					step: 'unzip',
+					zipPath: '/data.zip',
+					extractToPath: '/wordpress',
+				},
+				{
+					step: 'rm',
+					path: '/data.zip',
+				},
+				{
+					step: 'runSqlFile',
+					path: '/wordpress/schema/_Schema.sql',
+				},
+				{
+					step: 'rm',
+					path: '/wordpress/wp-content/mu-plugins/1-show-admin-credentials-on-wp-login.php',
+				},
+			];
+
+			if(pluginUrl && pluginName)
+			{
+				steps.push(...[
+					{
+						step: 'writeFile',
+						path: '/plugin.zip',
+						data: {
+							'resource': 'url',
+							'url': pluginUrl,
+						},
+					},
+					{
+						step: 'unzip',
+						zipPath: '/plugin.zip',
+						extractToPath: '/wordpress/wp-content/plugins',
+					},
+					{
+						step: 'rm',
+						path: '/plugin.zip',
+					},
+					{
+						step: 'activatePlugin',
+						pluginName: pluginName,
+						pluginPath: '/wordpress/wp-content/plugins/' + pluginName,
+					},
+				]);
+			}
+
+			steps.push(...[
+				{
+					step: 'login',
+					username: username,
+					password: fakepass,
+				}
+			]);
+
+			const client = await startPlaygroundWeb({
+				iframe: document.getElementById('wp-playground'),
+				remoteUrl: `https://playground.wordpress.net/remote.html`,
+				blueprint: {
+					preferredVersions: {
+						wp: <?=json_encode(COLLECTOR_WP_VERSION);?>,
+						php: <?=json_encode(COLLECTOR_PHP_VERSION);?>,
+					},
+					phpExtensionBundles: ['kitchen-sink'],
+					steps,
+				},
+			});
+
+			client.isReady().then(() => client.goTo('/wp-admin/plugins.php'));
```

Comment:
This is an `async` function so you could do `await client.isReady()`

Reply:
Hint: This link points to the `files` tab and it's showing this:

<img width="1160" alt="CleanShot 2023-12-11 at 23 03 36@2x" src="https://github.com/WordPress/playground-tools/assets/205419/20e784d3-a826-467c-a66c-99977b67215c">

To share a specific change in a way that can be previewed even after the PR evolves, you typically want to link to a particular commit.